archive.tf

```diff
@@ -19,3 +19,18 @@ resource "null_resource" "archive" {
     command = "${lookup(data.external.archive.result, "build_command")}"
   }
 }
+
+# Check that the null_resource.archive file has been built. This will rebuild
+# it if missing. This is used to catch situations where the Terraform state
+# does not match the Lambda function in AWS, e.g. after someone manually
+# deletes the Lambda function. If the file is rebuilt here, the build
+# output is unfortunately invisible.
+data "external" "built" {
+  program = ["python", "${path.module}/built.py"]
```

Comment:
Please specify python2 instead of just python here.

Reply:
I'll leave that change for #5